MediaBrowser.Model/Dto/BaseItemDto.cs

```diff
@@ -111,21 +111,21 @@ public class BaseItemDto : IHasProviderIds, IItemDto, IHasServerId
         /// Gets or sets the external urls.
         /// </summary>
         /// <value>The external urls.</value>
-        public ExternalUrl[] ExternalUrls { get; set; }
+        public IReadOnlyList<ExternalUrl> ExternalUrls { get; set; }
```

Comment:
same here. Those fields get serialised in the backend and AFAIK the json serializer cannot handle those interfaces well